ci/build-all-steps.yml

```diff
@@ -56,6 +56,11 @@ steps:
     displayName: Stage package
     condition: and(succeeded(), ne(variables['build.reason'], 'PullRequest'), ne(variables['numTasks'], 0))
 
+  # Sign all task zips as nuget packages
```

Comment:
This should probably have the same criteria as publishing does where we check that it's not a PR and there are tasks.

Reply:
good catch!

Reply:
I added the condition inside the template since templates inclusions don't have "conditions"